flowtorch/bijectors/utils.py

```diff
@@ -0,0 +1,50 @@
+import functools
+
+_RECORD_FLOW = True
```

Comment:
Will this work for multi-threaded code?

Reply:
Good point. Do you mean MT or MP? For MP, I guess it depends what you do in main and what you do on the worker. If the worker has his own context manager that should be ok, but i don't think it will propagate through main -> workers.
For MT I'm not sure I don't use that very often in python.

Reply:
Yep I mean multi-threaded. How about we leave it for now and see if there's a better solution to global state if it becomes a problem?